### PR TITLE
Update changelog for 0.7.0 release

### PR DESCRIPTION
**Pull request by will-moore**
I added PR numbers this time (as we do for OMERO.figure) since I think this is useful if anyone wants to know more, particularly on the main announcement, e.g. https://www.openmicroscopy.org/2018/08/28/figure-4-0-1.html

See changelog at https://github.com/will-moore/omero-iviewer/blob/changelog_0.7.0/CHANGELOG.md

### CODE REVIEW

**Comment by sbesson** [comment]
Is the goal to make these links clickable ? In this case, you might neeed to use the Markdown syntax - see e.g. https://raw.githubusercontent.com/ome/omero-cli-render/master/CHANGES.md

**Comment by jburel** [comment]
Few PRs will need to be added to the changelog

* https://github.com/ome/omero-iviewer/pull/234
* https://github.com/ome/omero-iviewer/pull/232
* https://github.com/ome/omero-iviewer/pull/229
* https://github.com/ome/omero-iviewer/pull/228

OL update implies a build system change. This should be indicated

**Comment by jburel** [comment]
We will do a pre-release build to test https://github.com/ome/omero-iviewer/pull/237 and also evaluate ROI support in IDR
so the version will be 0.7.0a1

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+0.7.0 (February 2019)
+---------------------
+
+This version includes:
+
+- handling large numbers of ROIs on image [#231](https://github.com/ome/omero-iviewer/pull/231)
+- drag and drop thumbnails to open image viewers (not supported on Internet Explorer) [#233](https://github.com/ome/omero-iviewer/pull/233)
+- support URL query parameters for x, y and zoom [#236](https://github.com/ome/omero-iviewer/pull/236)
+- update to use latest version of OpenLayers [#218](https://github.com/ome/omero-iviewer/pull/218)
```
Comment (jburel) [inline CHANGELOG.md]: mention version

**Pull request merged**